tests/mod_api/test_controller.py

```diff
@@ -756,21 +761,32 @@ def setUp(self):
                                     content_type='application/json')
         data = json.loads(response.data)
         self.jwt = data['token']
+        self.auth = "bearer %s" % self.jwt
 
     @patch('app.mod_api.models.BitStore.change_acl')
     @patch('app.mod_api.models.MetaDataDB.change_status')
     def test_return_200_if_all_goes_well(self, change_status, change_acl):
         change_acl.return_value = True
         change_status.return_value = True
-        response = self.client.delete(self.url)
+
+        response = self.client.delete(self.url, headers=dict(Authorization=self.auth))
         self.assertEqual(response.status_code, 200)
 
     @patch('app.mod_api.models.BitStore.change_acl')
     @patch('app.mod_api.models.MetaDataDB.change_status')
+    def test_return_403_not_allowed_to_do_operation(self, change_status, change_acl):
```

Comment:
The test should check that any of those users are denied: Anonymous, Viewer, Other publisher editor